src/site/filters/liquid.js

```diff
@@ -441,16 +441,24 @@ module.exports = function registerFilters() {
     return null;
   };
 
+  // TODO - add unit tests for this!
   liquid.filters.deriveLastBreadcrumbFromPath = (
     breadcrumbs,
     string,
     currentPath,
+    replaceLastItem = false,
   ) => {
     const last = {
       url: { path: currentPath, routed: true },
```

Comment:
What's the `routed: true` property for?

Reply:
Not sure, but we didn't change it in the PR.

Reply:
It's a cms thing. It means Drupal has created a more human readable path alias for the page.